scene/property_list_helper.cpp

```diff
@@ -77,11 +77,11 @@ void PropertyListHelper::get_property_list(List<PropertyInfo> *p_list, int p_cou
 			const Property &property = E.value;
 
 			PropertyInfo info = property.info;
-			if (_call_getter(property.getter, i) == property.default_value) {
+			info.name = vformat("%s%d/%s", prefix, i, info.name);
+
+			if (_call_getter(property.getter, i) == object->property_get_revert(info.name)) {
```

Comment:
I don't get why property that equals to its revert value (so you can't revert it) is still saved in the scene and has to be handled here :/